src/services/aragon-backend/queries/use-daos.ts

```diff
@@ -63,24 +65,27 @@ const fetchDaos = async (
 };
 
 export const useDaos = (
-  params: IFetchDaosParams,
-  options: UseInfiniteQueryOptions<IPaginatedResponse<IDao>> = {}
+  initialParams: IFetchDaosParams,
+  options: Omit<
+    UseInfiniteQueryOptions<IPaginatedResponse<IDaoPages>>,
+    'queryKey' | 'getNextPageParam' | 'initialPageParam'
+  >
 ) => {
-  return useInfiniteQuery(
-    aragonBackendQueryKeys.daos(params),
-    ({pageParam}) => fetchDaos({...params, ...pageParam}),
-    {
-      ...options,
-      getNextPageParam: (lastPage: IPaginatedResponse<IDao>) => {
-        const {skip, total, take} = lastPage;
-        const hasNextPage = skip + take < total;
+  return useInfiniteQuery({
+    queryKey: aragonBackendQueryKeys.daos(initialParams),
+    queryFn: ({pageParam = initialParams}) =>
```

Comment:
A default value shouldn't be needed, react-query should set this as the `initialPageParam` property
```suggestion
    queryFn: ({pageParam}) =>
```